app/src/main/java/gov/va/vro/security/ApiAuthKeyManager.java

```diff
@@ -14,10 +16,12 @@
 @Slf4j
 @NoArgsConstructor
 @AllArgsConstructor
+@Order(1)
```

Comment:
Let's add a comment here explaining why we need to set the order. Is there a clashing Authentication Manager?